receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -170,10 +171,16 @@ func (mg *metricGroup) toDistributionPoint(orderedLabelKeys []string, dest *pmet
 	}
 	point.SetTimestamp(tsNanos)
 	populateAttributes(orderedLabelKeys, mg.ls, point.Attributes())
-
+	mg.setExemplars(exemplarsdata, point.Exemplars())
 	return true
 }
 
+func (mg *metricGroup) setExemplars(exemplarMap map[float64]pmetric.Exemplar, exemplars pmetric.ExemplarSlice) {
```

Comment:
nit: this doesn't need to pass the exemplarMap as an arg, since you can just use `mg.exemplars`.  That should remove the need to pass it through other functions (e.g. toNumberDataPoint) as well.